infra/base-images/base-runner/run_fuzzer

```diff
@@ -114,6 +114,9 @@ if [[ "$FUZZING_ENGINE" = afl ]]; then
   test -e "$OUT/afl_cmplog.txt" && AFL_FUZZER_ARGS="$AFL_FUZZER_ARGS -l 2 -c $OUT/$FUZZER"
   # If $OUT/afl++.dict we load it as a dictionary for afl-fuzz.
   test -e "$OUT/afl++.dict" && AFL_FUZZER_ARGS="$AFL_FUZZER_ARGS -x $OUT/afl++.dict"
+  # Ensure timeout is a bit large than 1sec as some of the OSS-Fuzz fuzzers
+  # are slower than this. 
+  AFL_FUZZER_ARGS="$AFL_FUZZER_ARGS -t 50000"
```

Comment:
that would be 50 seconds - a tad too much?;)

Reply:
The thought process was in essence all or nothing, i.e. if the harness actually runs I thought it should get in and the plan was then to push the decision of timeouts over the Clusterfuzz due to the reasoning Abhishek highlight above.

Reply:
IMHO it makes no sense to have seeds that take longer than 5 seconds to complete. rather add the '+' so that afl-fuzz doesnt abort but skips longer running ones.
I think `-t 6000+` is maybe the best choice?

Reply:
Yeah I agree on that for sure. I have just tried simply adding `-t 1000+` and it seems to be working fairly well for the issue cases. Will update the PR in just a sec.

Reply:
I set it to `-t 5000+`